discli

```diff
@@ -82,6 +96,9 @@ class DiscourseHelper:
             }, f, default_flow_style=False)
         print("Values set: {}".format(self.identity))
 
+    def forget(self):
```

Comment:
Nice addition. I know we don't support it now, but I'm imaging a future where there may be multiple discourses that the user has logged into.